frontend/src/app/not-found.js

```diff
@@ -1,7 +1,5 @@
-"use client";
-
 import { redirect } from "next/navigation";
 
 export default function NotFound() {
-  redirect("/en");
+  redirect("/en/not-found");
```

Comment:
Since translation is used on the 404 page, can we reuse the initial translation if available? Otherwise, default to '/en' if the path is '/<random-path>' (without a translation path).